doc/source/cluster/kubernetes/user-guides/persist-kuberay-custom-resource-logs.md

```diff
@@ -191,14 +191,15 @@ kind: RayCluster
 metadata:
   name: raycluster-fluentbit-sidecar-logs
 spec:
-  rayVersion: '2.9.0'
+  rayVersion: '2.47.1'
   headGroupSpec:
-    rayStartParams: {}
+    # Before KubeRay 1.4.0, rayStartParams is required. It's optional after that.
+    # rayStartParams: {}
```

Comment:
commented out `rayStartParams` with a comment and used latest ray version to make it less likely user uses incompatible autoscaler image